magento/magento2
# #1797: The vertical image is not centered in Category image preview

### Description (*)

This PR will introduce the fix to align the position of vertical images to centre in category image preview.
### Related Pull Requests

N/A
### Fixed Issues (if relevant)

1. Fixes https://github.com/magento/adobe-stock-integration/issues/1797

### Manual testing scenarios (*)

1. A vertical image saved to Media gallery
_To save a vertical image to Media Gallery, perform the following steps:_
- Open Search Adobe Stock
- Click Filters
- Select Orientation - Vertical
- Apply Filters
- Save Preview of one the found Vertical images
2. Go to Catalog - Categories
3. Expand Content and click Select from Gallery
4. Select the previously saved Vertical image and click Add Selected
5. The preview image should be centered.
### Questions or comments


### Contribution checklist (*)
